### PR TITLE
use missinglink

### PR DESCRIPTION
**Pull request by honnix**
# Hey, I just made a Pull Request!

## Description

According to https://github.com/spotify/flo/pull/180#issuecomment-429744658

## Checklist for PR author(s)

- [x] Changes are covered by unit test
- [x] All tests pass
- [ ] Code coverage check passes
- [ ] Error handling is tested
- [ ] Errors are handled at the appropriate layer
- [ ] Errors that cannot be handled where they occur are propagated
- [ ] Relevant documentation updated
- [x] This PR has NO breaking change to public API
- [ ] This PR has breaking change to public API and it is documented

## Checklist for PR reviewer(s)

- [ ] This PR has been incorporated in release note for the coming version
- [ ] Risky changes introduced by this PR have been all considered

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -8,7 +8,11 @@ jobs:
       - checkout
       - restore_cache:
           key: v1-deps-{{ checksum "pom.xml" }}
-      - run: mvn -B dependency:go-offline compile test-compile
+      - run:
+          environment:
+            # Maven heap size
+            MAVEN_OPTS: -Xmx256m -Xms256m
```
Comment (danielnorberg) [inline .circleci/config.yml]: ⭐️

Comment (honnix) [reply .circleci/config.yml]: Still not working. Will check later.

Comment (honnix) [reply .circleci/config.yml]: `[ERROR] GC overhead limit exceeded -> [Help 1]` not enough now

**Comment by kanterov** [comment]
If it works locally, but not in CI, it doesn't have to be in CI. Could be just put into a profile and verified manually before release.

**Comment by danielnorberg** [comment]
👍

**Pull request merged**